<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UseCard</class>
 <widget class="QWidget" name="UseCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="drawPileButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/game/resource/ui/deck.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="energy">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/game/resource/ui/energyRed.png);</string>
   </property>
   <property name="text">
    <string>3/3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="discardPile">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/game/resource/ui/discard.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exhaustPile">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>89</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/game/resource/ui/exhaust.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="endTurnButton">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>9</y>
     <width>100</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/game/resource/ui/endturn.png);
color: rgb(255, 255, 255);
font: 700 11pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>结束回合</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>880</width>
     <height>215</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>880</width>
      <height>215</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>-11</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/game/resource/ui/endturn.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>-11</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/game/resource/ui/endturn.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
